.github/workflows/test.yml

```diff
@@ -17,22 +17,22 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        # TODO: fix lsp-sonarlint-c++-reports-issues on macos and include it
+        os: [ubuntu-latest]
```

Comment:
Can we ignore specific tests instead of entirely removing the `macos-latest`? 🤔

Reply:
Indeed, that's better. I ignored just `lsp-sonarlint-c++-reports-issues` on macos